app/views/user/show.erb

```diff
@@ -1,6 +1,7 @@
 <div class="container">
-  <section class="page-header">
-  <h1><%= profile.username %></h1>
+  <section class="row page-header">
+    <div class="col-md-1 main-profile-img"><img src="https://avatars.githubusercontent.com/u/980783?s=90" /></div>
```

Comment:
@tejasbubane I did some looking into the code, and I think something like:

```
<%= gravatar_tag current_user.avatar_url, size: 90 %>
```

would work ([based on nav/account.erb](https://github.com/exercism/exercism.io/blob/master/app/views/application/nav/account.erb#L3)).